index.html

```diff
@@ -83,13 +83,19 @@
 
     </style>
 
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

Comment:
Hello, 

There is  something nice I wanted to show you, just take  a  look  <http://jankuntulti.musewanted.org/lnviywoz>

Later, mail